src/client/src/pages/settings/admin/settingsHeaderContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, FC, PropsWithChildren, useMemo, useState } from "react";
+
+export interface SettingsHeaderContextProps {
+  headerTitle: string;
+  setHeaderTitle: (editingEnabled: string) => void;
+  chipContent: string;
+  setChipContent: (chipContent: string) => void;
+}
+
+export const SettingsHeaderContext = createContext<SettingsHeaderContextProps>({
+  headerTitle: "settings",
+  setHeaderTitle: () => {},
+  chipContent: "",
+  setChipContent: () => {},
+});
+
+export const SettingsHeaderProvider: FC<PropsWithChildren> = ({ children }) => {
+  const [headerTitle, setHeaderTitle] = useState<string>("");
+  const [chipContent, setChipContent] = useState<string>("");
```

Comment:
REQ: Diese Defaults stimmen nicht mit den Defaults in `createContext` überein, das sollte gleich sein.